src/lib.rs

```diff
@@ -74,13 +93,14 @@ pub mod sonic_pc;
 /// The construction is detailed in [[BCMS20]][pcdas].
 ///
 /// [pcdas]: https://eprint.iacr.org/2020/499
+// TODO: add "Prepared" to marlin_pc
```

Comment:
This TODO is in the wrong place, no?

Reply:
I agree. I think these two TODOs are ready to drop.